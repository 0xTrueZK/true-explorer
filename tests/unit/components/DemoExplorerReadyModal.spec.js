import MockHelper from '../MockHelper';

import DemoExplorerReadyModal from '@/components/DemoExplorerReadyModal.vue';

let helper;
beforeEach(() => {
    jest.clearAllMocks()
    helper = new MockHelper();
});

describe('DemoExplorerReadyModal.vue', () => {
    it('Should display link to demo explorer', async () => {
        const wrapper = helper.mountFn(DemoExplorerReadyModal, {
            data() {
                return {
                    dialog: true,
                    domain: 'http://my.explorer.com',
                }
            }
        });

        expect(wrapper.html()).toMatchSnapshot();
    });
});
