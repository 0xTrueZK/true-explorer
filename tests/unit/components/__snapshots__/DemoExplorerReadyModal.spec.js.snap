// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DemoExplorerReadyModal.vue Should display link to demo explorer 1`] = `
<div class="v-dialog__container">
  <div role="dialog" aria-modal="true" class="v-dialog__content v-dialog__content--active" style="z-index: 202;">
    <transition-stub name="dialog-transition" appear="true">
      <div tabindex="0" class="v-dialog v-dialog--active v-dialog--persistent" style="transform-origin: center center; max-width: 800px;">
        <div class="v-card v-sheet v-sheet--outlined theme--light">
          <div class="v-card__title success--text"><i aria-hidden="true" class="v-icon notranslate success--text mr-2 mdi mdi-check-circle theme--light"></i>
            Your explorer is ready!
          </div>
          <div class="v-card__text">
            You can access it here: <a href="//http://my.explorer.com" target="_blank">http://my.explorer.com</a></div>
        </div>
      </div>
    </transition-stub>
  </div>
</div>
`;
